pkgs/development/libraries/libxkbcommon/default.nix

```diff
@@ -2,8 +2,7 @@
 , xkeyboard_config, libxcb, libxml2
 , python3
 , libX11
-# To enable the "interactive-wayland" subcommand of xkbcli:
-, withWaylandSupport ? false, wayland, wayland-protocols
```

Comment:
We need to keep this option and can enable it by default on linux.

Reply:
Are you sure libxkbcommon is needed (or even works) on darwin?

Reply:
I don't know but gtk3 requires it and ofborg fails to eval right now. Maybe it is not required here https://github.com/NixOS/nixpkgs/blob/a726c366e7b130e08eb25c1e90dc0ab3ae8b64a6/pkgs/development/libraries/gtk/3.x.nix#L114 but I don't plan to test this right now.